oslo_vmware/rw_handles.py

```diff
@@ -558,6 +560,10 @@ def close(self):
         """
         try:
             self._release_lease()
+        except exceptions.ManagedObjectNotFoundException:
```

Comment:
This extends `exceptions.VimException`, so the below `except` would catch this as well.
Is this change then only for having a different log message?